u19_pipeline/ephys.py

```diff
@@ -48,17 +48,35 @@ class SkullReference(dj.Lookup):
 
 
 @schema
-class EphysSession(dj.Manual):
+class EphysRecording(dj.Computed):
     definition = """
     # General information of an ephys session
-    -> acquisition.Session
+    -> recording.Recording
     ---
-    ephys_directory: varchar(255)      # the absolute directory where the ephys data for this session will be stored in bucket
     """
+    @property
+    def key_source(self):
+        return recording.Recording & {'recording_modality': 'ephys'}
+
+    def make(self, key):
+        self.insert1(key)
+
+
+@schema
+class EphysSorting(dj.Computed):
+    definition = """
+    -> EphysRecording
+    -> recording.RecordingProcess
+    -----
+    """  
+
+    def make(self, key):
+        self.insert1(key)
+
 
 
 # ephys element requires table with name Session
-Session = EphysSession
+Session = EphysSorting
```

Comment:
Should we set `Session` to `EphysRecording` or `EphysSorting`?  Not sure what the correct answer is.